week1/homework/src/commands.js

```diff
@@ -0,0 +1,25 @@
+
+function whatURL(command, state) {
```

Comment:
This function name is slightly confusing especially since it returns state.  What would be a better name?